mx.truffleruby/suite.py

```diff
@@ -264,11 +277,23 @@
             "distDependencies": [
                 "truffle:TRUFFLE_API",
                 "truffle:TRUFFLE_NFI",
+                "truffleruby:TRUFFLERUBY-LAUNCHER",
```

Comment:
This looks good.
Could you check with `jar tf .../foo.jar | grep Main` that the launcher `.class` files are only in the truffleruby-launcher.jar?